tripleblindmarket/private-kit
# [SAF-665] Delete location history option

#247 ## Description:
This PR depends on delete secure storage feature not yet implemented!

Added an option to delete user data in the settings menu.

#### Screenshots:

<img src="https://user-images.githubusercontent.com/51460478/87043127-e7998380-c1f4-11ea-82dd-5b2c92595371.png" width="500">
<img src="https://user-images.githubusercontent.com/51460478/87043143-ecf6ce00-c1f4-11ea-83a7-91c1f0783099.png" width="500">

#### How to test:

- Open settings

- Click "Delete Location History

- Cancel / Confirm
